docs/source/getting-started/installing.adoc

```diff
@@ -66,6 +66,7 @@ If you encounter issues related to the hypervisor, see the xref:../using/trouble
 [NOTE]
 ====
 - On Windows operating system, due to issue link:https://github.com/minishift/minishift/issues/236[#236], you need to execute the minishift binary from the drive containing your *%USERPROFILE%* directory.
+The %USERPROFILE% directory must be located in your C:\ drive, and cannot be on a network drive.
```

Comment:
The last part is confusing where you are just saying `and cannot be on a network drive.` . Because it is true for any drive other than `C:` drive. Do you mean C drive can not be a network drive to make it work?